plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/TestOracleConnectorTest.java

```diff
@@ -15,21 +15,28 @@
 
 import com.google.common.collect.ImmutableMap;
 import io.airlift.testing.Closeables;
+import io.trino.Session;
 import io.trino.testing.QueryRunner;
 import io.trino.testing.sql.SqlExecutor;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.Test;
 
+import static io.trino.plugin.jdbc.ColumnWithAliasFormatter.DEFAULT_COLUMN_ALIAS_LENGTH;
+import static io.trino.plugin.jdbc.JdbcMetadataSessionProperties.JOIN_PUSHDOWN_ENABLED;
 import static io.trino.plugin.oracle.TestingOracleServer.TEST_PASS;
 import static io.trino.plugin.oracle.TestingOracleServer.TEST_SCHEMA;
 import static io.trino.plugin.oracle.TestingOracleServer.TEST_USER;
+import static io.trino.testing.TestingNames.randomNameSuffix;
 import static java.lang.String.format;
 import static java.util.stream.Collectors.joining;
 import static java.util.stream.IntStream.range;
 
+@Test(singleThreaded = true)
```

Comment:
why singleThreaded? Deserves a comment if we want to keep this.

Reply:
This is a left-over from when I had a @Before/@After method and another test to do the misconfiguration path. Will remove now. Thanks for spotting.